canonical/lxd-ui
# Redirect /ui/:project to the instance list for that project [WD-3071]

## Done

- Made the `/ui/:project` path redirect to the instance list.

Fixes [WD-3071](https://warthogs.atlassian.net/browse/WD-3071)

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - Visit any `/ui/:project` path of an existing project
    - Check that it correctly redirects to the instance list for that project

[WD-3071]: https://warthogs.atlassian.net/browse/WD-3071?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
